src/common/checksum.c

```diff
@@ -409,7 +410,9 @@ daos_csummer_csum_compare(struct daos_csummer *obj, uint8_t *a,
 	if (C_TRACE_ENABLED()) {
 		C_TRACE("Comparing: ");
 		daos_csummer_trace_csum(obj, a);
+		C_TRACE("\nWith: ");
 		daos_csummer_trace_csum(obj, b);
+		C_TRACE("\n: ");
```

Comment:
the related multiple lines log msg possibly mess up when the func is called concurrently by multi-process/threads.

Reply:
Ya, I'll address this in another patch.